lib/check_basic_abilities.rb

```diff
@@ -77,14 +77,11 @@ def extra_perms_for_all_users
     # 2) it's related to at least one public team
     # 3) it's related to a private team which the @user has access to
     can :read, [Account, ProjectMedia, Source], user_id: [@user.id, nil]
-    can :read, [Source, Media, Link, Claim], projects: { team: { private: false }}
```

Comment:
Do we really want to remove this permission?

Reply:
Yes as this permission for source depend on `Projects` which we removed so I got an error `undefined method projects for Source object`

Reply:
sounds good thanks